wyvern/components/features/feature_retrieval_pipeline.py

```diff
@@ -160,8 +160,6 @@ async def execute(
                 **kwargs,
             )
         )
-        current_request = request_context.ensure_current_request()
-        current_request.feature_map = feature_retrieval_response
```

Comment:
This is here so that realtime features will have access to the batch features. so we should keep it here